pkg/load/load_test.go

```diff
@@ -574,6 +577,102 @@ func TestConfig(t *testing.T) {
 	}
 }
 
+func correctHandler(t *testing.T, jsonConfig []byte) http.Handler {
```

Comment:
These should be inlined into the test cases you are using them in, as they are validating specific test data

Reply:
Fixed